ngraph_bridge/ngraph_tensor_manager.h

```diff
@@ -80,17 +101,23 @@ class NGraphTensorManager {
   int m_number_of_outputs;
 
   // Book-keeping for weights-on-device optimizations
+  // indexes wrt all inputs/outputs
   vector<int> m_input_indexes_from_variables;
   vector<int> m_output_indexes_assigning_variable;
   vector<int> m_output_indexes_that_need_copy;
 
-  // All indexes that are not for from/to variables
+  // All indexes that are not from/to variables
```

Comment:
I wont be able to change all the names in this PR. I can make changes to atmost  2-3 members + functions. Making changes to all can be taken in a separate PR. 

**Some Explanation**
There are only two data structures that are relative to pipelined inputs rather that all inputs. they have similar names and start with `GetPipelinedInputIndexes` to denote that we are asking relative to Pipelined Indexes (At least that's what i thought). The header also documents the indexes are wrt to what. May be this will help and is sufficient.

GetPipelinedInputIndexesThatArePrefetched() / m_pipelined_input_indexes_prefetched
GetPipelinedInputIndexesThatAreNotPrefetched() / m_pipelined_input_indexes_not_prefetched

Reply:
It might not be as straight forward to understand for a person reading the code for the first time. I would still suggest to change whatever you can in this PR to something more descriptive.

Reply:
I can change these two and their corresponding functions. Will wait for others suggestions before making the change
m_prefetched_in_indxs_from_pipelined_ins
m_not_prefetched_in_indxs_from_pipelined_ins

Reply:
Another suggestion:
GetPipelinedInputIndexesThatArePrefetched() / m_pipelined_input_indexes_that_are_prefetched
GetPipelinedInputIndexesThatAreNotPrefetched() / m_pipelined_input_indexes_that_are_not_prefetched